cloud/migrate-mst-cloud/index.md

```diff
@@ -0,0 +1,44 @@
+# Migrate from Managed Service for TimescaleDB to Timescale Cloud
+You can migrate your data from Managed Service for TimescaleDB to Timescale
+Cloud. This allows you to use Timescale Cloud's exclusive features, including
+separate scaling for compute and storage requirements, first-class multi-node
+support, and more.
+
+There are two methods for migrating your data:
+
+*   [Migrate your entire database at once][migrate-entire]: This method
+    directly transfers all data and schemas, including TimescaleDB-specific
+    features. Your hypertables, continuous aggregates, and policies are
+    automatically available in Cloud.
+*   [Migrate your schema and data separately][migrate-separately]: With this
+    method, you migrate your tables one by one for easier failure recovery. If
+    migration fails mid-way, you can restart from the failure point rather than
+    from the beginning. However, TimescaleDB-specific features won't be
+    automatically migrated. Follow the instructions to restore your hypertables,
+    continuous aggregates, and policies.
+
+## Choose a migration method
+Which method you choose depends on your database size, network upload and
+download speeds, and tolerance for failure recovery.
+
+If your database is smaller than FIXME&nbsp;GB, choose to migrate your entire
+database at once. You can also migrate larger databases using this method, but
+the copying process must keep running, potentially over days or weeks. If the
+copy is interrupted, the process needs to be restarted. If you think an
+interruption in the copy is possible, choose to migrate your schema and data
+separately instead.
```

Comment:
@lukas-bernert put this back in after reading Evan's comment in Slack. Do you have a reasonable size estimate for the cutoff? (in GB, number of rows, or anything)

Reply:
For now I'd suggest 100GB (compressed or uncompressed). this should take them ~10-15 hours to dump and 10-15h to restore.
We can run some benchmarks in the future to give a better time estimate.

Reply:
though I wouldn't include any time estimate for now
these are totally arbitrary